starter_code/express-cinema-project/bin/seeds.js

```diff
@@ -0,0 +1,86 @@
+const mongoose = require("mongoose");
+const Movie = require("../models/movie.js")
```

Comment:
be careful, your file is called Movie.js (capital M), it won't find it if you don't write it like that : 
const Movie = require("../models/Movie.js")